test/utils/general.test.ts

```diff
@@ -90,38 +90,3 @@ describe('snakeCaseToCamelCase()', () => {
         });
     });
 });
-
-describe('parseContentTypeFromResponse', () => {
```

Comment:
i dont mind moving the method, but i would rather not loose any tests we already have - are those cases covered in the http crawler tests somehow?

Reply:
Yes 100% of them